recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -555,7 +562,10 @@ install:
               done
             fi
 
-            $SUDO $KUBECTL apply -f $KUBECTL_YAML_DIR/newrelic-k8s.yml
+            if [[ $($SUDO $KUBECTL apply -f $KUBECTL_YAML_DIR/newrelic-k8s.yml) ]]; then
+            else
+              exit 131
```

Comment:
I would suggest adding an echo "ERROR = $? from $SUDO $KUBECTL apply -f $KUBECTL_YAML_DIR/newrelic-k8s.yml" on the failure path so that we can understand the reason for the failure.

Reply:
Same reasoning for choosing to not set a custom echo as I mentioned in the comment above.